UnusualCombat.tex

```diff
@@ -11,35 +11,37 @@ \part*{\begin{center}
 }
 \section{Item interactions \small (PHB p.190)}
 You can interact with one item or feature of the environment for free, during your movement or action.
-A second interaction requires an action.\\
+A second interaction requires an action.
+\vspace{3mm}
+
 Examples:
 \begin{itemize}
-\item draw / sheathe weapon
-\item open door / pull lever / extinguish small flame
-\item take an item from a container / pick up an item
-\item stuff something into your mouth / drink something
-\item hand someone something / accept something from someone
-\end{itemize} 
+\item Draw or sheathe your weapon
+\item Open a door, pull a lever or extinguish a small flame
+\item Take an item from a container or pick it up
+\item Stuff something into your mouth or drink something
+\item Hand someone something or accept something from someone
+\end{itemize}
 \section{Dropping prone \small (PHB p.190 + 292)}
 \begin{itemize}
-\item You can drop prone for free
+\item You can drop prone without any costs
```

Comment:
The wording "for free" is consistent with the PHB, gives just as much information and just makes more sense